code/modules/modular_computers/file_system/programs/system/file_browser.dm

```diff
@@ -144,6 +151,27 @@
 			return 1
 		var/datum/computer_file/C = F.clone(0)
 		HDD.store_file(C)
+	if(href_list["PRG_encrypt"])
+		. = 1
+		var/obj/item/weapon/computer_hardware/hard_drive/HDD = computer.hard_drive
+		if (!HDD)
+			return
+		var/datum/computer_file/F = HDD.find_file_by_name(href_list["PRG_encrypt"])
+		if (!F)
+			return
```

Comment:
It shouldnt be possible to encrypt / decrypt files if they are undeletable.
-> Those are system files that arnt meant to be interacted with.